features/mbedtls/mbed_lib.json

```diff
@@ -0,0 +1,122 @@
+{
+    "name": "mbedtls",
+    "config": {
+        "APP_CONFIG_FILE": {
```

Comment:
Why all these macros are written in upper case?

Reply:
I figured since the generated #defines were uppercase, they should be. Looking at other examples, I see they are not. I'll change to lowercase.